### ISSUE TITLE
PackageReference Condition warning

### ISSUE THREAD

**Issue by nietras** [mhutch/MonoDevelop.MSBuildEditor#199]
We often have to selective condition on package references especially native runtime packages like shown below. Hence, the warning is in that case not particularly relevant.

![image](https://github.com/mhutch/MonoDevelop.MSBuildEditor/assets/10798831/05768cac-3e59-4426-b5e1-4cddfa115a84)

**Comment by mhutch** [comment]
Yes, the current implementation of this analyzers is very naive and will result in a lot of false positives. Fixing it properly will require property dependency tracking, which is some ways off. For now, I could change it to check for `$(Configuration)` only.

Once we have editorconfig and the ability to suppress individual analyzers I might consider re-enabling it in its current form, as I think "hey, this might be problematic, suppress if you're sure you know what you're doing" has some value.
